static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -99,12 +100,25 @@ function useReplayData({
   const projects = useProjects();
 
   const api = useApi();
+  const queryClient = useQueryClient();
 
   const [state, setState] = useState<State>(INITIAL_STATE);
   const [attachments, setAttachments] = useState<unknown[]>([]);
   const attachmentMap = useRef<Map<string, unknown[]>>(new Map()); // Map keys are always iterated by insertion order
   const [errors, setErrors] = useState<ReplayError[]>([]);
-  const [replayRecord, setReplayRecord] = useState<ReplayRecord>();
+
+  // Fetch every field of the replay. We're overfetching, not every field is used
```

Comment:
This is just moved code, but might be helpful to include why we're fetching fields.